UPGRADE-7.0.md

```diff
@@ -55,6 +55,12 @@ Support for Adobe Flash was removed. We recommend to search the media manager fo
 extensions "flv", "f4v" and "swf") and remove them.
 The media manager will also display where these files are still used; these usages should also be removed.
 
+## Remove module_cms_search Table
+
+The CMS search module was removed. If the project still needs this module, restore it from an earlier Chameleon release.
+Otherwise remove the associated database table by calling `TCMSLogChange::deleteTable('module_cms_search')` in an update
```

Comment:
What about the template module record?

Reply:
Added an update that removes the module :-)